Human density import

samaperrin opened the pull request:
# Why have changes been made?

We need human activity data. SSB has [detailed human density ](https://www.ssb.no/natur-og-miljo/geodata) and building density data, which has been implemented in this push.

# What changes have been made?
- functions/import/environment/get_ssb.R - New function to download SSB data based on exact covariate
- pipeline/import/utils/defineEnvSource.R - New section to prepare parameters for get_ssb
- pipeline/models/utils/biodiversityMetricEstimation.R - Debug biodiv. estimation when <2 red-list species
- data/external/focalCovariates.csv - New rows for human covariates
- README.md - Quick explainer of temp folder - this should be used for large files (that we want to avoid downloading multiple time) used to derive environmental variables
- Have also added the package "fasterize" to help convert the shp files into rasters

Commit:
Addition of human density variables

Commit:
Added ssb as a possible data source

Commit:
Added new import function for SSB human data

Commit:
Added fasterize package

Commit:
Update to README to reflect purpose of temp folder

RRTogunov reviewed (commented):
Current version works for me. Just have to check how function handles getRutenett = FALSE, and might need to pop define reference raster outside of function in case aspect isn't downloaded.

RRTogunov committed:
debug biodiv.MetricEst. when <2 redlist spp.
wrapped biodiv estimator with if statement to
skip if 0 redlist spp, or return single vector if 2 redlist spp.
Previously, rowSums would return error.

Commit:
Removed saving of red listed predictions - ultimately unnecessary

samaperrin commented:
I made one extra change - I was saving the red listed metric as a file which would later be overwritten. I removed this redundancy.
